kairon/chat/handlers/channels/voice.py

```
@@ -0,0 +1,140 @@
+        fallback = config.get("reprompt_fallback_phrase",
+                              "I'm sorry, I didn't get that. Could you please repeat?")
+        try:
+            agent = AgentProcessor.get_agent(self.bot)
+            tracker = await agent.tracker_store.retrieve(sender_id)
+            if tracker:
+                last = next((e for e in reversed(tracker.events)
+                             if isinstance(e, BotUttered)), None)
+                if last and last.text:
+                    return [last.text]
+        except Exception:
+            pass
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)